docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html

```diff
@@ -14,40 +14,40 @@
       current_url_path = window.location.pathname;
 
       if (startsWith(current_url_path, "/ansible-core/")) {
-        document.write('<div id="banner_id" class="admonition caution">');
+        document.write('<div id="core_banner_id" class="admonition caution">');
```

Comment:
This wasn't really necessary because only one of the code branches produces a banner. It was only problematic for the branch which produces two `<div>`s with the same ID. But never mind, it'll be easier with my theme refactoring.